### ISSUE TITLE
[FxA] FF is stuck in Create a Firefox Account after selecting the "How old are you?" field

### ISSUE THREAD

**Issue by abodea** [mozilla-mobile/firefox-tv#2607]
## Steps to reproduce
1. Launch FF.
2. Add any email that is not bound to a FF account.
3. Tap Continue.
4. Tap on the up button from the remote and select `How old are you?` field.
5. Tap up and down on the remote in order to select any other option from the Create a Firefox Account screen.

### Expected behavior
The rest of the options can be selected.
### Actual behavior
FF is stuck on the Create a Firefox Account screen when `How old are you?` field is selected.
### Device information
* Fire TV device: **`Fire TV Stick 4k and 2nd Gen Stick`**
* Latest affected Firefox version: **`Firefox TV v4.1 LAT2`**
* Earliest affected Firefox version: **`Firefox TV v4.1 LAT2`**
* Turbo Mode: **`On and Off`**

![20190726_153806](https://user-images.githubusercontent.com/42831109/61952057-85cc1380-afbb-11e9-916a-af599de5f319.jpg)

**Comment by mcomella** [comment]
@Baron-Severin Did you test the create account flow? If not, can you file a bug to go through it and report issues to FxA, like the other bug? Then raise it to Ashley as something to do sooner rather than later.

**Comment by athomasmoz** [comment]
Yep, here's the FxA bug https://github.com/mozilla-mobile/firefox-tv/issues/2610

**Comment by athomasmoz** [comment]
@abodea This has been fixed in FxA #2610 , can you retest?

**Comment by npark-mozilla** [comment]
This is still broken in 4.2 LAT-2
